### ISSUE TITLE
508-defect-1 ❗️ Launchblocker: Important links missing on Health Care Services

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21726]
Label: 20132

    - [508-defect-1 :exclamation: Launchblocker]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1

Issue Title: Health Care Services missing links
 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a user, I expect that all promised resources are properly linked.

<h2>Details</h2>
The following link texts on the Health Care services page are not linked to anything:
<ul>
<li>Learn more about our pharmacy</li>
<li>Learn more and connect with a care coordinator (under mental health care/behavioral health)</li>
<li>Learn more and connect with a care coordinator under suicide prevention</li>
<li>Learn more and connect with a support coordinator (under caregiver support)</li>
<li>Learn more and connect with a care coordinator under "homeless veterans,"</li>
<li>Learn more and connect with a care coordinator (Under LGBT veterins)</li>
<li>Learn more and connect with a care coordinator (under minority veterans)</li>
<li>Learn more and connect with a patient advocate</li>
<li>Learn more and connect with a coordinator (Under returning service member care post 911...)</li>
<li>Learn more and connect with a care coordinator (Under women veterans)</li>
</ul>

Acceptance Criteria

Environment

Steps to Recreate

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs
